test/Go/TransitiveSpec.hs

```diff
@@ -0,0 +1,160 @@
+module Go.TransitiveSpec (spec) where
+
+import Control.Algebra (run)
+import DepTypes
+import GraphUtil
+import Strategy.Go.Transitive
+import Strategy.Go.Types (graphingGolang)
+import Test.Hspec
+
+spec :: Spec
+spec = spec_packageToModule
+
+-- HACK(fossas/team-analysis#514) see Strategy.Go.Transitive for more details
+--
+-- Ensure that packages are correctly mapped to their modules, and that packages
+-- aren't dropped from the dependency graph
+spec_packageToModule :: Spec
+spec_packageToModule =
+  describe "normalizeImportsToModules" $ do
```

Comment:
Can you add a separate test in here that checks that not-module packages are correctly handled? I know this is already covered by `should map package imports to their modules`, but I prefer explicitly encoding this intent.